src/main/resources/swagger/rawls.yaml

```diff
@@ -3289,6 +3286,19 @@ definitions:
         type: ''
         description: Map[String,Attribute]
 
+  AttributeEntityReference:
+    description: ''
+    required:
+      - entityName
```

Comment:
Nitpick: will output these in the reverse order so ideally the docs should match